test/README.md

```diff
@@ -38,9 +39,11 @@ can automatically fix the test so that it matches the new output (if that is the
 right behaviour!) or it can mark it as broken for you.
 
 ## Unit tests
+
 To run unit tests, just use `ninja -C build test` (or `meson test -C build`)
 from the top directory (replace `build` with the name of the directory you used
 to build Rizin).
+You can run one specific testcase (e.g. `bin`) using `meson test -C build bin`.
```

Comment:
Probably a "testcase category" instead?

Reply:
i think is ok there. let's see what the others thinks about it.

Reply:
EIther way is fine for me.

Reply:
I am with @XVilka, "testcase category" is better imo